src/main/keyboard/shortcutHandler.js

```diff
@@ -110,6 +110,16 @@ class Keybindings {
       ['tabs.cycle-backward', 'CmdOrCtrl+Shift+Tab'],
       ['tabs.switch-to-left', 'CmdOrCtrl+PageUp'],
       ['tabs.switch-to-right', 'CmdOrCtrl+PageDown'],
+      ['tabs.switch-to-first', 'Alt+1'],
```

Comment:
@Jocs @MrHeer It `Cmd` used on macOS by other applications?

Reply:
not very clear